docs/topics/asset/giveaway/multiple.md

```diff
@@ -0,0 +1,114 @@
+# Asset Multiple Giveaway Smart Contract
+
+## Introduction
+
+Users can earn the assets in many ways like discord hunt, unicly contest and so on.
+Here admin lock their earned assets in this contract and users can claim them with matching params.
+
```

Comment:
The reader needs a bit of context.
Explain what is an asset (find inspiration here https://sandboxgame.gitbook.io/the-sandbox/assets/what-are-assets)
Explain the concept of giveaway
and then the way of earning assets

Your last sentence shouldn't be in the introduction